src/run/mod.rs

```diff
@@ -167,7 +167,13 @@ impl Run {
         self.category_name.push_str(name.as_ref());
     }
 
-    /// Sets the path of the associated splits file in the file system.
+    /// Gets the path of the associated splits file in the file system.
+    #[inline]
+    pub fn get_path(&mut self) -> &Option<PathBuf> {
```

Comment:
Can we name this just `path`? `get_` as a prefix is not idiomatic.

Reply:
Certainly